packages/beacon-node/src/network/processor/gossipHandlers.ts

```diff
@@ -168,7 +170,12 @@ export function getGossipHandlers(modules: ValidatorFnsModules, options: GossipH
               peerRpcScores.applyAction(peerIdFromString(peerIdStr), PeerAction.LowToleranceError, "BadGossipBlock");
           }
         }
-        logger.error("Error receiving block", {slot: signedBlock.message.slot, peer: peerIdStr}, e as Error);
+        metrics?.gossipBlock.processBlockErrors.inc({blockErrorCode});
+        logger.error(
+          "Error receiving block",
+          {slot: signedBlock.message.slot, peer: peerIdStr, blockErrorCode, seenTimestampSec},
```

Comment:
The error code is already logged in the first line of this log

Reply:
Updated.